plugins/BEdita/Core/config/bootstrap.php

```diff
@@ -60,10 +60,16 @@
 
 /**
  * Use custom DateType & DateTimeType
+ *
+ * Timezone conversion is not currently supported in CakePHP 3.8 saving to Database
+ * We should use `timezone` as defined in `Datasource` configuration, with 'UTC' as default
+ * For now is safe to force 'UTC' as timezone
+ *
+ * See https://github.com/cakephp/cakephp/issues/13646
  */
 Type::set('date', new DateType());
-Type::set('datetime', new DateTimeType());
-Type::set('timestamp', new DateTimeType());
+Type::set('datetime', (new DateTimeType())->setTimezone('UTC'));
```

Comment:
Maybe we could slightly improve this setting server timezone
```php
Type::set('datetime', (new DateTimeType())->setTimezone(date_default_timezone_get()));
```
but anyway I think this solution is ok for now.

Reply:
You're right, but the timezone setting should probably come from datasource connection config.

What about a constant like:

```php
if (!defined('BEDITA_DEFAULT_TIMEZONE')) {
    define('BEDITA_DEFAULT_TIMEZONE', 'UTC');
}
```
or better an env var

```
->setTimezone(env('BEDITA_DEFAULT_TIMEZONE', 'UTC'));
```